keyboards/massdrop/ctrl/keymaps/endgame/keymap.c

```diff
@@ -126,6 +110,56 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
     */
 };
 
+#ifdef _______
+#undef _______
+#define _______ {0, 0, 0}
```

Comment:
This is causing the keymap to fail to compile.

Reply:
ummm, that's weird, it compiles fine here. Just tried it.

Reply:
This is fixed now, it was a syntax issue in the led config map. Some merge fail.